<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_error</class>
 <widget class="QDialog" name="dialog_error">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Error</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/Ico.ico</normaloff>../assets/Ico.ico</iconset>
  </property>
  <widget class="QTextBrowser" name="error_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>560</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
